tests/output/test_io_function.py

```diff
@@ -33,6 +33,9 @@ def _get_mesh(cell_type, comm):
     elif cell_type == "quadrilateral":
         mesh = Mesh(join(cwd, "..", "meshes", "unitsquare_unstructured_quadrilaterals.msh"),
                     name=mesh_name, comm=comm)
+    elif cell_type == "hexahedral":
+        mesh = Mesh(join(cwd, "..", "meshes", "cube_hex.msh"),
```

Comment:
Check this is really unstructured.

Possibly also put in a Möbius strip.

Reply:
Found that `cube_hex.msh` has a vertex that is shared by 26 cells.

Added `möbius_solid.msh` and added a test in `tests/output/test_io_function.py`.

![Screenshot from 2022-11-17 20-06-42](https://user-images.githubusercontent.com/46749170/202742024-dce394ba-ab17-44ad-8fb0-567501cca738.png)